### ISSUE TITLE
Query Regarding Classification Accuracy

### ISSUE THREAD

**Issue by sumeetssaurav** [DhruvJawalkar/yoga-pose-estimation#2]
Hii,
Just have a little query regarding classification accuracy. Did you simply fine tuned the existing pre-trained resnet34 model. It seems little difficult to achieve such an accuracy since the images are visually similar.
Could you please explain if I am wrong?

Thank You

**Comment by DhruvJawalkar** [comment]
What exactly is your question?

**Comment by sumeetssaurav** [comment]
I mean, did you simply fine-tuned on ImageNet pre-trained resnet34 model using your yoga dataset with 107 classes??

I am also trying to fine-tune ImageNet pre-trained resnet50 model on your dataset, but not getting good accuracy.

**Comment by DhruvJawalkar** [comment]
Try to unfreeze all layers and train, resnet50 would have the capacity to overfit the training dataset.

**Comment by sumeetssaurav** [comment]
BTW 97 % is the training or validation accuracy. I did not find any split information in the dataset.

**Comment by aakash2016** [comment]
Hello there, I would really appreciate if you could suggest how we can achieve that level of accuracy. 🙂, What is the magic? I tried training both freezed and unfreezed resenet34. But am not able to achieve that level of acc. I am working with your dataset itself. But, trying to train myself and see the results. Any Help is appreciated. Thanks.

**Comment by paramaggarwal** [comment]
@sumeetssaurav what is your source for the `97%` number?
